src/renderer/angular/controllers/guest/import/keystore-controller.js

```diff
@@ -98,6 +98,7 @@ function GuestImportKeystoreController($rootScope, $scope, $log, $q, $timeout, $
         });
 
         unlockExistingWalletPromise.then((data) => {
+            console.log('YES')
```

Comment:
@altninja can you remove this logging unless it is needed so it would be good to have more explanatory message. Cheers

Reply:
oops missed one thanks!